src/main/java/gg/projecteden/nexus/features/listeners/Sleep.java

```diff
@@ -35,11 +35,13 @@
 
 				if (sleeping >= needed && sleepingWorlds.get(world) != State.SKIPPING)
 					skipNight(world);
-				else if (sleepingWorlds.get(world) == State.SLEEPING)
+				else if (sleepingWorlds.get(world) == State.SLEEPING) {}
```

Comment:
Whats with the empty brackets?

Reply:
Leftover from a stupid debugging thing, silly me - removed